code-challenges.js

```diff
@@ -41,11 +100,68 @@ var testString2 = "jabberwocky"
 // Expected output: "jAbBeRwOcKy"
 
 
+//The function must take in strings
 
+const capitalized = (string) => {
+  //have to split the string into array
+  let stringSplitter = string.split("")
+  //map through array and return odd index values capitlaized
+  let mappedArray = stringSplitter.map((value,index) => {
+    if(index % 2 == 0){
+      return value
+    } else {
+      return value.toUpperCase()
+    }
+  })
+  //must join array
+  return mappedArray.join("")
+}
+//conditional statement looking at odd and even index numbers to return the capital letters
+//must break string down into an array in order to interact with each elment of the array
+
+console.log(capitalized(testString1));
+console.log(capitalized(testString2));
 
 
 // --------------------5) Create a function that takes in two arrays as arguments and returns one array with no duplicate values. STRETCH: Use the spread operator to pass in a dynamic number of arguments.
 
 var arr1 = [3, 7, 10, 5, 4, 3, 3]
 var arr2 = [7, 8, 2, 3, 1, 5, 4]
 // Expected output: [3, 7, 10, 5, 4, 8, 2, 1]
+
+
+
+
+//make one long array using concat
+//this will help with all the manipulation happening in one array. need to find method that gets rid of duplicate values
+//have to set value
+
+const noDuplicates = (array1, array2) => {
+ let concatArray = array1.concat(array2)
+ //I knew it was going to most likley be a filter because it would have to be a shorter array or iterate thorught some other way but had hard time finding a way to only return first occurance of an element
+ let timesChecker = concatArray.filter((value, index) => {
+   return concatArray.indexOf(value) === index
+ })
+ return timesChecker
+}
+
+
+//This solution betllow is really cool but i dont really know why it worked. I found the solution online when trying to find out how to remove duplicates from an array.
+
+
+const noDuplicates2 = (array1, array2) => {
+ let concat2 = array1.concat(array2)
+ let noDup= [...new Set(concat2)]
+return noDup
+}
```

Comment:
I really really like that you have 2 totally different solutions here that approach this problem in very different ways. nice work